bitnami/moodle/values.yaml

```diff
@@ -58,7 +58,7 @@ moodleUsername: user
 ## Defaults to a random 10-character alphanumeric string if not set
 ## ref: https://github.com/bitnami/bitnami-docker-moodle#configuration
 ##
-moodlePassword: ""
+moodlePassword: password
```

Comment:
Please don't change the default values.

Reply:
this was an accident when I was doing some testing.